packaging/windows/install_script.nsis

```diff
@@ -0,0 +1,278 @@
+Unicode true
+# This installs two files, agent-windows-amd64.exe and logo.ico, creates a start menu shortcut, builds an uninstaller, and
+# adds uninstall information to the registry for Add/Remove Programs
+
+!define APPNAME "Grafana Agent"
+!define DESCRIPTION "The Grafana Agent collects observability data and sends it to a compatible remote write endpoint"
+# These will be displayed by the "Click here for support information" link in "Add/Remove Programs"
+!define HELPURL "https://github.com/grafana/agent/discussions" # "Support Information" link
+!define UPDATEURL "https://github.com/grafana/agent/releases" # "Product Updates" link
+!define ABOUTURL "https://github.com/grafana/agent" # "Publisher" link
+!include LogicLib.nsh
+
+RequestExecutionLevel admin #Require admin rights on NT6+ (When UAC is turned on)
+
+InstallDir "$PROGRAMFILES\${APPNAME}"
+
+# This will be in the installer/uninstaller's title bar
+Name "${APPNAME} ${VERSION}"
+Icon "logo.ico"
+outFile "/app/grafana-agent-installer.exe"
+
+!include LogicLib.nsh
+!include nsDialogs.nsh
+!include FileFunc.nsh
+
+
+# Everything must be global Vars
+
+# These are handles to dialogs
+Var Dialog
+Var RemoteWriteLabel
+Var RemoteWriteText
+
+Var RemoteWriteUserLabel
+Var RemoteWriteUserText
+
+Var RemoteWritePasswordLabel
+Var RemoteWritePasswordText
+
+Var EnableExporterLabel
+Var EnableExporterCheck
+
+# These are strings that hold the values
+
+Var RemoteUrl
+Var RemoteUser
+Var RemotePassword
+Var EnableExporterValue
+
+# Silent install params
+
+Var PassedInParameters
+
+Page directory
```

Comment:
Can we add a license page here with the Apache 2 license? Copying it to the install directory would be a good idea too IMO.

Reply:
Can we add a link or do you want the actual text to be on the page?